frontend/src/routes/(authenticated)/user/+page.ts

```diff
@@ -5,8 +5,21 @@ import type {
 
 } from '$lib/gql/types';
 import { getClient, graphql } from '$lib/gql';
-import { invalidate } from '$app/navigation';
-import {refreshJwt} from '$lib/user';
+import { goto, invalidate } from '$app/navigation';
+import { refreshJwt } from '$lib/user';
+import type { PageLoad } from './$types';
+import { browser } from '$app/environment';
+
+export const load = (async ({ url }) => {
+  if (browser) {
+    if (url.searchParams.has('verifiedEmail')) {
+      await goto(`${url.pathname}`, {
```

Comment:
is it better to call goto? or throw a redirect here? I'm not sure I just wanted to check.

Reply:
Actually...I should probably just remove this line.

Reply:
but if we navigate somewhere else what does it matter what is returned?

Reply:
I navigate to the same page (to clear the query params) and, because I `await` the navigation, the thing I return wins.